packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -55,9 +66,32 @@ export default function BookmarkFeedLayout({
       <FeedPageHeader className="mb-5">
         <h3 className="font-bold typo-callout">Bookmarks</h3>
       </FeedPageHeader>
-      <CustomFeedHeader className="relative mb-6">
+      <CustomFeedHeader className="flex relative mb-6">
```

Comment:
I think the relative class became obsolete because we no longer use absolute positioning inside right?

Reply:
You are right, I just tested without and it behaves correctly.